recipes/elemental-core/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "elemental-core" %}
+{% set version = "0.2.1.dev0" %}
```

Comment:
Right now, there is not. This is a component of a side project that gets attention whenever I can give it. I was hoping to use conda-forge to reduce my time spent configuring CI solutions and increase development time on the actual product.

However, I completely understand if conda-forge prefers packages that have reached a certain stability. Did I miss this in the documentation somewhere? If so, I sincerely apologize for adding to your already considerable maintenance burden.

What level of stability is conda-forge aiming to publish? I would be happy to hold off submitting packages until they reach that level.